saasbuilder-docker-compose.yaml

```diff
@@ -54,6 +56,20 @@ services:
         modifiable: true
         required: true
         export: false
+      - key: smtpHost
```

Comment:
lets add default values for both smtpHost and smtpPort as smtp.gmail.com and 587